### PR TITLE
tweak a test to compile on latest Scala 2.13 nightlies

### LINKED ISSUE #2355: RepresentableSuite.scala doesn't compile in Scala 2.13 community build
```
[cats] Running "test" in: testsJVM
[cats] [info] Compiling 88 Scala sources to /Users/tisue/community-builds.213/target-0.9.12/project-builds/cats-f6470f629c8fd47a8d425f3562a7e38f7d16ca33/tests/.jvm/target/scala-2.13.0-pre-ba03b6d/test-classes ...
[cats] [error] /Users/tisue/community-builds.213/target-0.9.12/project-builds/cats-f6470f629c8fd47a8d425f3562a7e38f7d16ca33/tests/src/test/scala/cats/tests/RepresentableSuite.scala:63:58: could not find implicit value for parameter iso: cats.laws.discipline.SemigroupalTests.Isomorphisms[[β$5$]String => β$5$]
[cats] [error]     checkAll("String => ?", MonadTests[String => ?].monad[String, String, String])
[cats] [error]                                                          ^
```

this is on 2.13.0-pre-ba03b6d

not sure what to make of this; the code compiles on 2.13.0-M4, so I suppose this is due to some post-M4 change in scala/scala

tracking this down could be tricky. 277 PRs (!) have been merged, as of scala/scala@ba03b6d, since M4, and Cats has a bunch of dependencies so it's not easy to build and test everything against 2.13.0-pre-ba03b6d

can someone who knows the cats code explain what might be going wrong here? and then I can ask the Scala team to take a look and see what PRs might explain it?

worst comes to worst, I can use the community build to bisect what Scala PR this regressed in, but doing that would take a long time

### PR DESCRIPTION
**Pull request by SethTisue**
fixes #2355

### CODE REVIEW

**Comment by SethTisue** [comment]
> did this fix the error?

yes

**Comment by ceedubs** [comment]
Do we have any insight into why this wasn't required before but is now? I'll admit to never understanding what these `Isomorphisms` are used for in tests. This change should probably go in, but it makes me wary of whether there's a bigger potential issue.

**Comment by kailuowang** [comment]
@ceedubs I myself am not familiar with the issue either. I left [this comment](https://github.com/typelevel/cats/issues/2355#issuecomment-409633558) on the original issue with a little bit that I know of.

File: tests/src/test/scala/cats/tests/RepresentableSuite.scala [hunk 1]
```diff
@@ -59,6 +59,7 @@ class RepresentableSuite extends CatsSuite {
   }
 
   {
+    implicit val isoPair = Isomorphisms.invariant[String => ?]
```
[the remark this answers is not in the document]
Comment (SethTisue) [reply tests/src/test/scala/cats/tests/RepresentableSuite.scala]: fixed

**Comment by ceedubs** [comment]
`¯\_(ツ)_/¯`

**Comment by ceedubs** [comment]
Thanks @SethTisue!

**Pull request merged**